common/scala/build.gradle

```diff
@@ -87,12 +87,16 @@ dependencies {
     compile 'io.reactivex:rxjava-reactive-streams:1.2.1'
     compile 'com.microsoft.azure:azure-cosmosdb:2.3.0'
 
+    //for mongo
+    compile 'org.mongodb.scala:mongo-scala-driver_2.12:2.6.0'
```

Comment:
i rebased this pr to master, and resolved conflicts.
in doing this, i picked up the latest version of the driver which might have been foolish.
please check it.